includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -336,43 +337,46 @@ public function data_table_header() {
 			$selected_course = (int) $_GET['filter_by_course_id']; // phpcs:ignore WordPress.Security.NonceVerification
 		}
 		?>
-		<div class="tablenav top">
-			<div class="alignleft actions bulkactions">
-				<div id="sensei-bulk-learner-actions-modal" style="display:none;">
-					<?php $this->render_bulk_actions_form( $courses ); ?>
-				</div>
-				<?php
-				echo wp_kses(
-					$this->render_bulk_action_select_box(),
-					array(
-						'option' => array(
-							'value' => array(),
-						),
-						'select' => array(
-							'id'   => array(),
-							'name' => array(),
-						),
-					)
-				);
-				?>
-				<button type="submit" id="sensei-bulk-learner-actions-modal-toggle" class="button button-primary action"><?php echo esc_html__( 'Select Courses', 'sensei-lms' ); ?></button>
+		<div class="sensei-student-bulk-actions__wrapper">
+			<div class="alignleft bulkactions sensei-student-bulk-actions__container">
+			<div id="sensei-bulk-learner-actions-modal" style="display:none;">
```

Comment:
I think we're missing a level of indentation here.

Reply:
Fix identation: [here](https://github.com/Automattic/sensei/pull/4997/commits/f5baa73cfac10ef5accc82ddb2a04606b654a11d) and [here](https://github.com/Automattic/sensei/pull/4997/commits/8c33acd4754f38cbc6ce99a6814ad5991b2106ef) (forgot one div )